### PR TITLE
BUGFIX: Avoid problem loading files in SimpleXML

### LINKED ISSUE #1598: Possible problem with xml php bug in \Neos\Flow\I18n\AbstractXmlParser::getRootNode
### Description

Possible problem with xml php bug in '\Neos\Flow\I18n\AbstractXmlParser::getRootNode'

grafik

### Steps to Reproduce

see https://bugs.php.net/bug.php?id=62577

#### Expected behavior

load xml file

#### Actual behavior

```
Packages/Framework/Neos.Flow/Classes/I18n/AbstractXmlParser.php: Parsing the XML file failed. These error were reported:
failed to load external entity "resource://Neos.Flow/Private/I18n/CLDR/Sources/main/root.xml" (line 0, column 0)
failed to load external entity "resource://Neos.Flow/Private/I18n/CLDR/Sources/main/root.xml" (line 0, column 0)
```

### Affected Versions

Flow: 3.x, 4.x, 5.x

You can get this information by running `composer show`.

### PR DESCRIPTION
**Pull request by kdambekalns**
Workaround for https://bugs.php.net/bug.php?id=62577

Fixes #1598

### CODE REVIEW

**Comment by kdambekalns** [comment]
Replaces #1599 and is based on 4.3 - thanks to @kaystrobach for the change!

File: Neos.Flow/Classes/I18n/AbstractXmlParser.php [hunk 1]
```diff
@@ -55,7 +55,7 @@ protected function getRootNode(string $sourcePath): \SimpleXMLElement
             throw new Exception\InvalidXmlFileException('The path "' . $sourcePath . '" does not point to an existing and accessible XML file.', 1328879703);
         }
         libxml_use_internal_errors(true);
-        $rootXmlNode = simplexml_load_file($sourcePath, 'SimpleXmlElement', \LIBXML_NOWARNING);
+        $rootXmlNode = simplexml_load_string(file_get_contents($sourcePath), 'SimpleXmlElement', \LIBXML_NOWARNING);
```
Comment (albe) [inline Neos.Flow/Classes/I18n/AbstractXmlParser.php]: Maybe add link to the simplexml bug to document why `load_file` isn't used?

Comment (kdambekalns) [reply Neos.Flow/Classes/I18n/AbstractXmlParser.php]: Isn't that what `git blame` is for? 😬

**Comment by albe** [comment]
Regarding travis failure, see #1601 
Still unsure why it worked up until at least 2 days ago

**Pull request merged**